.prospector.yaml

```diff
@@ -0,0 +1,5 @@
+pep257:
```

Comment:
Yes, it is to avoid Codacy complaing about things which are a bit annoying and (my opinion) debatable like empty lines sorrounding docstrings (D202, D203) and the fact that in an __init__.py you import but not use (D104)